doc: added information how to install test dependencies for each actor (#507)
oamg/leapp: docs/source/unit-testing.md

That is useful when you are debugging using printing to standard output.

To execute unit tests of all actors from all Leapp repositories in the
`leapp-repository` GitHub repository, first install test dependencies for each

> actor by running  `make install-deps` and then issue `make test` in the root

Reviewer: ```suggestion
actor by running  `make install-deps` followed by `make test` in the root
```
